src/gui/torrentcontentmodelitem.cpp

```diff
@@ -38,7 +39,7 @@ TorrentContentModelItem::TorrentContentModelItem(TorrentContentModelFolder *pare
     : m_parentItem(parent)
     , m_size(0)
     , m_remaining(0)
-    , m_priority(prio::NORMAL)
+    , m_priority(static_cast<int>(BitTorrent::FilePriority::Normal))
```

Comment:
Seems like it would be better to have `m_priority` and all related methods (here and in child classes) of type FilePriority.
I don't insist that you change it in this PR if it's too much for you.